final_score_and cat_images_disabled_when_user_click_next

mustafadurmusNl opened the pull request:
i updated the final score, i also updated displaying cat images  to disable them  when user click next button. I also updated background colour of correct answer to green when user click a wrong answer

mustafadurmusNl committed:
final_score_and cat_images_disabled_when_user_click_next

i updated the final score, i also updated displaying cat images  to disable them  when user click next button. I also updated background colour of correct answer to green when user click a wrong answer

mustafadurmusNl committed:
Merge branch 'main' into mustafaDurmus-questions_and_score

mustafadurmusNl committed:
skip_button_displays_correct_answer_as_green_background

keremilhan commented on public/style.css:
```diff
@@ -211,7 +211,7 @@ button {
 
 .skipped-correct { 
-  background-color: gray;
+  background-color: green;
```
- When the user skips the question, lets make the background blue instead of green.

mustafadurmusNl committed:
Merge branch 'main' into mustafaDurmus-questions_and_score

mustafadurmusNl committed:
skip-button-makes-answer -as blue-background

just a single update in css to make background blue when user click skip button
